packages/icons/src/index.ts

```diff
@@ -9,7 +9,7 @@ export { UxIconMap };
 
 export function configure(config: FrameworkConfiguration, icons?: Array<{name: string, material: string}>) {
   config.globalResources(PLATFORM.moduleName('./ux-icon'));
-  if (Array.isArray(icons) && icons.length > 0 && typeof icons[0].name === 'string' && typeof icons[0].material === 'string') {
-    UxIconMap.registerIcons(icons);
+  if (Array.isArray(icons) && icons.length > 0) {
+    Container.instance.get(UxIconMap).registerIcons(icons);
```

Comment:
Instead of using `Container.instance` we should use `config.container`. The `FrameworkConfig` instance will return the container instance for the current app, which is better to use in this case.

Reply:
Of course, using `Container` report the static issue to the Container class. Thanks for the feedback.